lib/filewatch/watch.rb

```diff
@@ -1,59 +1,36 @@
 require "logger"
+require_relative 'watched_file'
+
 if RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/
   require "filewatch/winhelper"
+  INODE_METHOD = :win_inode
+else
+  INODE_METHOD = :nix_inode
 end
 
+MAX_OPEN_FILES = ENV.fetch("MAX_OPEN_FILES", 4095).to_i
```

Comment:
But it needs a test and a better name because its in the global namespace and it may be overwritten by something else loading later.